docs/installation.md

```diff
@@ -2,233 +2,286 @@
 
 ## Overview
 
-This document walks you through the installation of the Zonemaster *Backend* in the following sections:
+Zonemaster *Backend* needs to run on an operating system. One can choose any of
+the following OS to install the *Backend* after having the required
+[Prerequisites](#prerequisites).
 
-1. [Choices](#choices) lets you make all choices up front that this guide presumes you've made.
-2. [Prerequisites](#prerequisites) lets you check all assumptions made by this guide about pre-installed software.
-3. [Dependencies](#dependencies) shows you how to install all software that your Zonemaster *Backend* will depend on.
-4. [Installation](#installation) shows you how to install the *Backend* itself.
-5. [Configuration](#configuration) shows you how to configure the *Backend*.
-6. [Startup](#startup) shows you how to start the *Backend*.
-7. [Post-installation sanity check](#post-installation-sanity-check) lets you verify that your *Backend* has started correctly.
+* [CentOS 7](#centos)  
+* [Debian 8(Jessie)](#debian--ubuntu)
+* [Ubuntu 16.04](#debian--ubuntu)
+* [FreeBSD](#freebsd) 

```

Comment:
It must be the same order as in the headline.
```
* [Debian 8(Jessie)](#ubuntu--debian)
* [Ubuntu 16.04](#ubuntu--debian)
```
I suggest that we point out FreeBSD 10.3 at this point (must be updated later).